### PR TITLE
feat: add erc7779 support

### PR DESCRIPTION
**Pull request by kopy-kat**
https://github.com/ethereum/ERCs/pull/663/files

### CODE REVIEW

**Comment by filmakarov** [comment]
added naive onRedelegation() implementation also

**Comment by kopy-kat** [comment]
think this looks good to go now - wdyt @filmakarov

File: src/core/ModuleManager.sol [hunk 1]
```diff
@@ -92,6 +95,40 @@ abstract contract ModuleManager is AccountBase, Receiver {
         IValidator(validator).onUninstall(disableModuleData);
     }
 
+    /*
+    function _tryUninstallValidators(bytes[] calldata data) internal {
```
Comment (livingrockrises) [inline src/core/ModuleManager.sol:99]: why is this left commented out?

File: src/core/ModuleManager.sol [hunk 2]
```diff
@@ -131,6 +168,40 @@ abstract contract ModuleManager is AccountBase, Receiver {
         IExecutor(executor).onUninstall(disableModuleData);
     }
 
+    /*
+    function _tryUninstallExecutors(bytes[] calldata data) internal {
```
Comment (livingrockrises) [inline src/core/ModuleManager.sol:172]: why is this left commented out?

**Pull request merged**